### ISSUE TITLE
Knex: Timeout acquiring a connection

### ISSUE THREAD

**Issue by Tealk** [louislam/uptime-kuma#4924]
### 📑 I have found these related issues/pull requests

https://github.com/louislam/uptime-kuma/issues/4598

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### Description

I have been getting the error message very frequently for 2 days:

> Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?

![grafik](https://github.com/louislam/uptime-kuma/assets/12276250/6e586ee2-85cb-4229-b9fa-3cb230c7b85a)

### 👟 Reproduction steps

I can't say, I've been running Kuma for months now without changing anything.

### 👀 Expected behavior

-

### 😓 Actual Behavior

-

### 🐻 Uptime-Kuma Version

1.23.13

### 💻 Operating System and Arch

Debian 12

### 🌐 Browser

Firefox

### 🖥️ Deployment Environment

- Runtime: Docker 26.1.4
- Database: sqlite
- Filesystem used to store the database on: ext4 on SSD
- number of monitors: 43

Installation: https://github.com/louislam/uptime-kuma/wiki/%F0%9F%94%A7-How-to-Install#-docker-compose

**Comment by CommanderStorm** [comment]
I don't know why you opened another issue while linking https://github.com/louislam/uptime-kuma/issues/4598.
The resolution of said issue has not changed.

I am going to close this as resolved by `v2.0` as I don't see evidence to the contrary.

> TL;DR:
> In v1, reducing either retention/ping-rate/.. or moving to a faster storage solution is the only option to get around this problem.
> In v2, we have likely resolved this problem. See #4500 for further details of what needs to happen before we can release this version.

**Comment by Tealk** [comment]
Because, as I said, it has been running on the VM for months without any changes and there were no problems and it suddenly started. The memory has not become slower.

**Comment by CommanderStorm** [comment]
Yes, that is the performance problem noted in https://github.com/louislam/uptime-kuma/issues/4598
I don't see a difference.
Retention is 90 days => maybe consider reducing this.

**Comment by CommanderStorm** [comment]
> I am going to close this as resolved by `v2.0` as I don't see evidence to the contrary.

Forgot to close as resolved

**Comment by Tealk** [comment]
is there a way to connect mysql when using docker compose? i can't find any references for this in the wiki.
https://github.com/louislam/uptime-kuma/wiki/Environment-Variables

**Comment by CommanderStorm** [comment]
Please see #4500 for further context on the v2.0 release

**Comment by Tealk** [comment]
the question was less whether it comes in the 2.x but whether I can connect it now with mysql.
